### PR TITLE
Fix/enhance animation

### PR DESCRIPTION
**Pull request by jbethanco**
Played around with this last night:
- Labels will animate movement to new positions on devices with iOS 14 and fade in out as they do currently on iOS 13
-Make CardLabel, for card view, to simplify animation, spacing, and alignment.
-Add WrappedNamespace file to wrap namespace call for geometry effect
-Change Landscape/Portrait calls to remove optional, and avoid unnecessary or repeated animations for non Landscape or Portrait orientations. (e.g. faceDown, faceUp, and unknown)

### CODE REVIEW

File: PBLogging/NewNavViews/Form781View/Overview/Form781Card.swift [hunk 1]
```diff
@@ -92,3 +92,25 @@ struct Form781Card_Previews: PreviewProvider {
...
+    let mutableText: String
+    var imagePadding: CGFloat = 0.0
+    
+    var body: some View {
+        HStack {
+            Image(systemName: image)
+                .padding(.horizontal, imagePadding)
+            Text(labelText)
+                .font(.pblRegular(size: 16))
+            Spacer()
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/Overview/Form781Card.swift:110]: I don't think we want a spacer here. There is a gap in the Figma, but the editable text is much closer to the associated label and this spaces out equal from the two labels in landscape.

**Review by Pete-Hoch — CHANGES_REQUESTED** [review]
Looks nice, but landscapes layout does not match Figma.

**Comment by jbethanco** [comment]
I think the fix is adjusting the card label frame width. I left it wider
than needed as wasn’t sure we were done playing with the fonts.

**Comment by jbethanco** [comment]
> Looks nice, but landscapes layout does not match Figma.

Think the width change fixed it.

**Review by jbethanco — COMMENTED** [review]
fixed

**Review by Pete-Hoch — APPROVED** [review]
Nice.

**Pull request merged**